### ISSUE TITLE
Names should be linked to the wikipedia article about them

### ISSUE THREAD

**Issue by kitwestneat** [kitwestneat/celeb#2]
One way would be to have a metadata store keyed on the name, or could just guess based on the name to start with.

**Comment by leggomuhgreggo** [comment]
Yeah diseases too probz. I think we could run our terms against either wikipedia or google's knowledge graph API, and iron out any bad matches that whatever "I'm feeling lucky" batch returns.

What db were you thinking?
